README.md

```diff
@@ -26,5 +26,5 @@ The specification is written in [Bikeshed](https://github.com/tabatkins/bikeshed
 ### Syntax changes should be filed against the following parsers:
 
 *   [widlparser](https://github.com/plinss/widlparser/issues/new) (used by Bikeshed)
-*   [WebIDL parser](https://github.com/w3c/webidl2.js/issues/new) (used by idlharness.js)
+*   [WebIDL parser](https://github.com/w3c/webidl2.js/issues/new/choose) (used by idlharness.js)
```

Comment:
Could we link directly to `https://github.com/w3c/webidl2.js/issues/new?assignees=&labels=syntax-update&template=syntax_update.md&title=BREAKING+CHANGE%3A+`?

Reply:
`/choose` is more stable IMO - no change required if default assignees, title etc. change in the template. Just at cost of one extra click.